tests/cli/workflow/test_logs.py

```diff
@@ -175,7 +175,7 @@ def test_download_dir_passed(
 
             # Custom mocks
             dumped_path = "<dumped path sentinel>"
-            action._dumper.dump.return_value = dumped_path
+            action._dumper.dump.return_value = (dumped_path,)
```

Comment:
**nitpick**: isn't this method expected to always return two paths?

Reply:
This better 055f8a7?